dedalus/tests/test_spherical3D_operators.py

```diff
@@ -225,20 +226,111 @@ def test_interpolate_scalar(Nphi, Ntheta, Nr, k, dealias, dtype, basis, radius):
 @pytest.mark.parametrize('dealias', dealias_range)
 @pytest.mark.parametrize('basis', [build_ball, build_shell])
 @pytest.mark.parametrize('dtype', [np.float64, np.complex128])
-@pytest.mark.parametrize('radius', [0.5, 1.0, 1.5])
-def test_interpolate_vector(Nphi, Ntheta, Nr, k, dealias, dtype, basis, radius):
+@pytest.mark.parametrize('theta_interp', [0, np.pi/4, np.pi/2, np.pi])
```

Comment:
What happens when we interpolate theta to 0 or pi?

Reply:
Right now the output remains a function of phi and r, and will just give identical values for all phi (but structurally still include all m's).